lib/content-services/src/lib/content-metadata/components/content-metadata/content-metadata.component.ts

```diff
@@ -214,6 +211,13 @@ export class ContentMetadataComponent implements OnChanges, OnInit, OnDestroy {
         return this._assignedTags;
     }
 
+    set tags(tags: string[]) {
+        this._tags = tags;
+        this.tagsToDisplay = this._tags.map((tag) => {
```

Comment:
eslint issue: 
<img width="1165" alt="image" src="https://github.com/user-attachments/assets/e22ae259-3ffe-4505-9c7b-5cbccc51cf1f">

Reply:
I don't have any issue there

Reply:
I think something wrong with your settings (for example IDE settings) because I can see that this rule is added in eslint so that eslint issue is valid and should be fixed. Note that it's warning, not error, so maybe you have some settings which skip warnings but it should be fixed as it's in our configuration for eslint
<img width="1279" alt="image" src="https://github.com/user-attachments/assets/ed298959-959a-44f9-92e3-458ef43956e7">

Reply:
I wonder if that can be something about IDE version, because it wasn't straight change and adding there braces wasn't intuitive